src/main/kotlin/com/novatecgmbh/eventsourcing/axon/web/exception_handling/CommandExecutionExceptionHandler.kt

```diff
@@ -0,0 +1,29 @@
+package com.novatecgmbh.eventsourcing.axon.web.exception_handling
```

Comment:
package names with understore are quite uncommon. I would rename this package to "error" or "exception". Furthermore, since this is cross functional concern, this package can be moved to a "common" package next the the "project" package (the one for the context). This will most likely be used by other contexts as well and is a candidate later for a library.